app/containers/MassEnergizeSuperAdmin/Community/EditCommunityByCommunityAdmin.js

```diff
@@ -361,6 +361,16 @@ const createFormJson = ({ community, autoOpenMediaLibrary }) => {
               ],
             },
           },
+          {
```

Comment:
Community Admins don't need to be able to change this variable.  That can be a superadmin only thing.
Please remove from this file.